src/main/java/com/byznass/tiolktrack/kernel/dao/UserProvider.java

```diff
@@ -1,8 +1,10 @@
 package com.byznass.tiolktrack.kernel.dao;
 
+import com.byznass.tiolktrack.kernel.TiolkTrackException;
 import com.byznass.tiolktrack.kernel.model.User;
 
 public interface UserProvider {
 
 	User getUser(String userId) throws NoUserWithSuchIdException;
+	boolean exists(String userId) throws TiolkTrackException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.kernel.TiolkTrackException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)